solace-spring-boot-autoconfigure/solace-java-spring-boot-autoconfigure/pom.xml

```diff
@@ -45,5 +49,10 @@
 			<version>1.19.0</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.18.30</version>
+		</dependency>
```

Comment:
* Remove the version. Lombok's version is already provided through the `spring-boot-dependencies` BOM.
* Give this the `provided` scope.